packages/server-wallet/src/protocols/defunder.ts

```diff
@@ -126,4 +153,30 @@ export class Defunder {
       tx
     );
   }
+
+  /**
+   * Assume that we are going to submit collaborative transaction(s) unless:
+   * 1. There is an agreed upon order of transaction submitters.
+   * 2. There is a transaction submitter before us who has funds in the channel.
+   *
+   * This method is public for unit testing only
```

Comment:
One way to accomplish this, while maintaining privacy, is making it a `protected` function that can be overridden as public by a `TestDefunder` class.

But, why is it a `static` method, instead of just a function?

Reply:
Good point! b3878c317